test/IntegrationTests/WcfNetFrameworkTests.cs

```diff
@@ -15,15 +15,36 @@
 // </copyright>
 
 #if NET462
+using IntegrationTests.Helpers;
 using Xunit.Abstractions;
 
 namespace IntegrationTests;
 
 public class WcfNetFrameworkTests : WcfTestsBase
 {
+    private const string ServiceName = "Wcf.Client.NetFramework";
+
     public WcfNetFrameworkTests(ITestOutputHelper output)
-        : base("Wcf.Client.NetFramework", output)
+        : base(ServiceName, output)
+    {
+    }
+
+    [Fact(Skip = "https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/1429")]
+    [Trait("Category", "EndToEnd")]
+    public async Task TracesResource()
```

Comment:
As we discussed, I would test with independent app.
It is not related to WCF,

Can we call method TracesResourceForAppConfig?

Reply:
I started to think if we should also test the `web.config` in `AspNetTests` as it is redundant, however it feels "safer" to double-check to avoid regression bugs. This is why I am leaving the `AspNetTests`.